settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "mt5": {
+        "login": 75319438,
+        "password": "DfN!5xYq",
```

Comment:
Let's add settings.json to our .gitignore file so that we can avoid pushing our credentials up to git and then include instructions on setting one up in our README (as well as a warning that code and tests won't work without it).

Reply:
Settings.json has been added to .gitignore, and README.md has been updated